simple-webrtc-test/tests/simple-rtc-datachannel.spec.js

```diff
@@ -0,0 +1,31 @@
+const { test, expect } = require('@playwright/test');
+
+const path = require('path');
+const fileURL = 'file://' + path.resolve(__dirname, './index.html');
+
+
+test.beforeEach(async ({page}) => {
+  await page.goto(fileURL);
+});
+
+test.afterEach(async ({page}) => {
+	const disconnectButton = page.getByTestId('disconnectButton');
+	await disconnectButton.click();
+});
+
+
+test.describe('Simple RTC DataChannel', () => {
+	test('should send a message from pc1 to pc2', async ({ page }) => {
+	    	const connectButton = page.getByTestId('connectButton');
+		const messageInputBox = page.getByTestId('message');
+		const sendButton = page.getByTestId('sendButton');
+		const receivedBox = page.getByTestId('receivebox');
```

Comment:
I don't understand. Are these referencing the same page? Are you not testing communication between two browsers?

Reply:
If the intention is to set up an RTCDataChannel between two browsers, wouldn't that essentially be a simplistic replication of [the test here](https://github.com/libp2p/js-libp2p/blob/master/packages/transport-webrtc/test/basics.spec.ts#L100)?

Reply:
Yes, that's the goal. Except without any js-libp2p. From the [original issue](https://github.com/libp2p/test-plans/issues/235#issue-1805319171):

> Maybe a very simple non-js-libp2p browser only interop test would be a good way to test if the browser is at fault.

Reply:
I understand, but upon reconsideration I wondering what insights that may yield given this test passes consistently. Do you think it would be fruitful to run that test in the multidim interop setup?

Reply:
Does that test run with multiple browsers? Looking at it, it seems like it's run within a single context.

Reply:
Gotcha, you are correct it does not run in multiple browsers.